src/Math/include/BipedalLocomotion/Math/ContactWrenchCone.h

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file ContactWrenchCone.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_MATH_CONTACT_WRENCH_CONE_H
+#define BIPEDAL_LOCOMOTION_MATH_CONTACT_WRENCH_CONE_H
+
+#include <memory>
+
+#include <Eigen/Dense>
+
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+
+
+namespace BipedalLocomotion
+{
+namespace Math
+{
+
+/**
+ * ContactWrenchCone computes the polyhedral approximation of the contact wrench friction cone.
+ * A surface remains fixed with respect the environment if the contact wrench lies in a cone
+ * described by
+ * \f[
+ * f ^c \cdot n > 0 \quad | f ^t | \le \mu f^c \cdot n
+ * \f]
+ *  where \f$ f^c \f$ is the
+ * contact force, \f$ n \f$ is the vector normal to the contact surface. \f$ f^t \f$ is the
+ * tangential force to the contact surface and \f$ \mu \f$ is the friction parameter. In addition we
+ * require CoP inside the support area and a bounded yaw torque. The ContactWrenchCone aims to
+ * compute the polyhedral approximation of \f$ | f ^t | \le \mu f^c \cdot n \f$ by spitting the base
+ * of the cone into slices while considering the contact torque constraints. The class implements
+ * the equations presented in [_Stability of Surface Contacts for Humanoid Robots: Closed-Form
+ * Formulae of the Contact Wrench Cone for Rectangular Support Areas_
+ * paper](https://ieeexplore.ieee.org/document/7139910). However differently from the original
+ * work, the origin of the frame attached to the contact surface is not required to be in the
+ * center of the surface.
+ * @note The ContactWrenchCone class express the constraints in body representation (so called right
+ * trivialization). Please check
```

Comment:
```suggestion
 * @note The ContactWrenchCone class express the constraints in body representation (so called left
 * trivialization). Please check
```

Body = left
Spatial = right